tests/test_consumer.py

```diff
@@ -110,3 +113,55 @@ def test_produce_replacement_messages(self):
             ("event-replacements", b"1", b'{"project_id":1}'),
             ("event-replacements", b"2", b'{"project_id":2}'),
         ]
+
+
+def test_streaming_consumer_strategy() -> None:
```

Comment:
This is a pretty pathetic test but it at least adds a good deal of coverage to the all of the moving pieces here.